service/lib/agama/cmdline_args.rb

```diff
@@ -51,20 +53,11 @@ def self.read_from(path)
         next unless option.start_with?(CMDLINE_PREFIX)
 
         key, value = option.split("=", 2)
-        key.gsub!(CMDLINE_PREFIX, "")
```

Comment:
Ouch, this could potentially delete more than expected! :scream: 

```
irb(main):001:0> "inst.rage.against.the.machine".gsub("inst.", "")
=> "rage.agathe.machine"
```